<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeGeneration</class>
 <widget class="QDialog" name="TreeGeneration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLabel" name="r1c1">
     <property name="styleSheet">
      <string notr="true">border: 6px solid black;
background: white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="r1c2">
     <property name="styleSheet">
      <string notr="true">border: 6px solid black;
background: white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="r2c0">
     <property name="styleSheet">
      <string notr="true">border: 6px solid black;
background: white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="gtdglbl">
     <property name="text">
      <string>Generate Tree Until Goal Node Generated</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="GenerateTreeDepth">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QPushButton" name="GenerateTreeGoal">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="r2c1">
     <property name="styleSheet">
      <string notr="true">border: 6px solid black;
background: white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QSpinBox" name="depthvalue"/>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="r2c2">
     <property name="styleSheet">
      <string notr="true">border: 6px solid black;
background: white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="gtcdlbl">
     <property name="styleSheet">
      <string notr="true">border{
1 px solid black;
}</string>
     </property>
     <property name="text">
      <string>Generate Tree of Certain Depth</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="r1c0">
     <property name="styleSheet">
      <string notr="true">border: 6px solid black;
background: white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="r0c0">
     <property name="styleSheet">
      <string notr="true">border: 6px solid black;
background: white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="r0c1">
     <property name="styleSheet">
      <string notr="true">border: 6px solid black;
background: white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="r0c2">
     <property name="styleSheet">
      <string notr="true">border: 6px solid black;
background: white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="instatelabel">
     <property name="styleSheet">
      <string notr="true">border: 5px solid white;
color: white;
background: black;</string>
     </property>
     <property name="text">
      <string>Initial State</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="randomize">
     <property name="text">
      <string>Randomize</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
